Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -279,13 +280,39 @@ void Processor::check_invoke_scheduler()
 NAKED void thread_context_first_enter(void)
 {
     asm(
-        // FIXME: Implement this
-        "wfi \n");
```

Comment:
Oh, you're removing this again in this later commit. Disregard my earlier comment about formatting in here then; not that important if it disappears again the same PR.

Reply:
Yep, was easier to make atomic commits this way.